src/org/jitsi/impl/neomedia/rtcp/RTCPSenderInfoUtils.java

```diff
@@ -134,4 +135,92 @@ public static long getNtpTimestampLSW(byte[] buf, int off, int len)
 
         return RawPacket.readInt(buf, off + 4, len) & 0xffffffffl;
     }
+
+    public static int setTimestamp(ByteArrayBuffer baf, long ts)
+    {
+        if (baf == null)
+        {
+            return -1;
+        }
+
+        return setTimestamp(baf.getBuffer(), baf.getOffset(), baf.getLength(), ts);
+    }
+
+    public static long getTimestamp(ByteArrayBuffer baf)
+    {
+        if (baf == null)
+        {
+            return -1;
+        }
+
+        return getTimestamp(baf.getBuffer(), baf.getOffset(), baf.getLength());
```

Comment:
The documentation in this class is incorrect, for example getTimestamp(byte[] buf, int off, int len) expects a buffer pointing to the start of an RTCP Sender Report (as opposed to the Sender Info structure that is found at an offset in the SR).